apps/prairielearn/src/lib/groups.ts

```diff
@@ -1,13 +1,16 @@
 import * as error from '@prairielearn/error';
-import { flash } from '@prairielearn/flash';
 import { z } from 'zod';
 import _ = require('lodash');
 
 import * as sqldb from '@prairielearn/postgres';
 import { idsEqual } from './id';
 import { GroupSchema, IdSchema, User, UserSchema } from './db-types';
+import { getEnrollmentForUserInCourseInstance } from '../models/enrollment';
+import { selectUserByUid } from '../models/user';
 const sql = sqldb.loadSqlEquiv(__filename);
 
+export class GroupOperationError extends Error {}
```

Comment:
It's considered best practice to set the `name` property for custom error instances.

```suggestion
export class GroupOperationError extends Error {
  constructor(message) {
    super(message);
    this.name = 'GroupOperationError';
  }
}
```